wiki/Help_Centre/Account_Restrictions/en.md

```diff
@@ -58,3 +79,16 @@ There are several reasons why an appeal might be denied.
 - **You have been evading or playing on other accounts.** Creating another account to be able to play online while your account is restricted is considered an attempt at evading the restriction and will reset your cooldown to three months from the date you created the account. If you attempt evading too many times, your appeal might be permanently denied. If you are still accessing or playing on your multi-accounts while restricted, your cooldown will also be reset. Remember that multi-accounting is against the [rules](/wiki/rules) and you are expected to only play on your original account. If you are unsure which account counts as your original account, send us an email and we will let you know!
 - **You continued offending while restricted.** There is no reason to unrestrict someone who continues to break the rules even after having been restricted. If you do continue offending, the cooldown for your account will be reset to your last offence. In the case of extreme or continuous offences, your appeal might be permanently denied.
 - **Your history is too severe.** In cases of extreme offenders, or players who get restricted again after they have already been unrestricted on a successful appeal, we might simply state that your history is too severe to rejoin the game at the current time. Should this apply to you, we recommend waiting **a long time** before appealing again, and making sure you do not break any further rules in that time.
+
+## Common restriction reasons and cooldowns
+
+|Restriction reason|Cooldown duration|Appeal notes|
+|-|-|-|
+|Judgement Error|No cooldown|If found erroneous, no impact on the account.|
+|Multi-account|Permanent (multis)|Main account will be left unrestricted, possibly silenced.|
+|Excessive multi-accounting|3 months|Appeal terms apply|
+|Account sharing|3 months|Appeal terms apply and partial rollback|
+|Cheating|3 months|Appeal terms apply and full rollback|
+|Excessive misconduct|Variable|Appeal terms apply. Cooldown will depend on history.|
+|Cheating/Sharing for a tournament|6 months| Appeal terms apply, permanent tournament ban|
+|Second offences|6 months|Appeal terms apply, permanent tournament ban|
```

Comment:
What about third offences and onwards?, is there something to note on those cases?

Reply:
Probably change Second offences to Subsequent offences.